llarp/lokinet_shared.cpp

```diff
@@ -1,4 +1,5 @@
 #include <lokinet.h>
+#include <filesystem>
```

Comment:
this should be `llarp/util/fs.hpp` as it contains compat shims for macos and such.